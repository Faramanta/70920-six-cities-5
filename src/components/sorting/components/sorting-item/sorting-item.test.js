import renderer from 'react-test-renderer';
import SortingItem from "./sorting-item";

describe(`SortingItem render correctly`, () => {
  it(`SortingItem isActive`, () => {
    const tree = renderer
      .create(
          <SortingItem
            onSortItemClick={() => {}}
            isActive={true}
          />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`SortingItem no active`, () => {
    const tree = renderer
      .create(
          <SortingItem
            onSortItemClick={() => {}}
            isActive={false}
          />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
