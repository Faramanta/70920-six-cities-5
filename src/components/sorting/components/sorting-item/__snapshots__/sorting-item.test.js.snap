// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortingItem render correctly SortingItem isActive 1`] = `
<li
  className="places__option places__option--active"
  onClick={[Function]}
  tabIndex="0"
/>
`;

exports[`SortingItem render correctly SortingItem no active 1`] = `
<li
  className="places__option "
  onClick={[Function]}
  tabIndex="0"
/>
`;
